djangoevents/utils.py

```diff
@@ -0,0 +1,52 @@
+import inspect
+from .domain import BaseEntity
+from .domain import BaseAggregate
+from .domain import DomainEvent
+
+
+def list_concrete_aggregates():
```

Comment:
This does not list all aggregates, only those already imported. How one should make sure all aggregates are loaded during `validate_event_schemas` command run? Is there some standard way to do that I am missing?